src/pages/RegisterPage/RegisterPage.tsx

```diff
@@ -7,7 +7,15 @@ import { Link } from 'react-router-dom';
 import getRoutedButtonLink from 'shared/utils/getRoutedButtonLink';
 import paths from 'shared/const/paths';
 
+import { useAppDispatch } from 'hooks';
+import { signupReguested } from 'store/signupReducers';
+
 function RegisterPage({ title }: GenericPageProps): JSX.Element {
+  const dispatch = useAppDispatch();
+  const onSubmit = () => {
+    dispatch(signupReguested());
+  };
```

Comment:
❗️
При передаче обработчика в качестве пропса, нужно использовать `useCallback`
```suggestion
  const onSubmit = useCallback(() => {
    dispatch(signupReguested());
  }, [dispatch]);
```